crates/release_plz_core/Cargo.toml

```diff
@@ -20,12 +20,16 @@ cargo_metadata.workspace = true
 cargo-clone-core.workspace = true
 cargo-edit = { workspace = true, features = ["set-version"] }
 chrono = { workspace = true, features = ["clock"] }
+clap = { workspace = true, features = ["derive"] }
```

Comment:
That's a good point, it's just used for the derive on `GitBackendKind`, would we be ok with just `clap-derive` as a dep? Otherwise I can move `GitBackendKind`

Reply:
Nevermind, looks like the derives don't work great without `clap` itself so I've moved `GitBackendKind` to be with the args struct